ral-facilities/datagateway
# Small changes to investigation card view #723

## Description
As per the issue:

- Changing "Name" to "Dataset" to reduce ambiguity of page results
- Changing color of hyperlink to orange in ISIS branding header to improve readability

The hyperlink color is the same as `ukri.bright.orange` in the UKRI theme. I decided to make it stay this color even after being `visited` as the default visited color is purple, which clashes with the blue background.

## Testing instructions
Navigate to an investigation card view such as `/browse/instrument/1/facilityCycle/14/investigation/87?view=card` to see changes

- [ ] Compare [SciGateway](https://scigateway-preprod.esc.rl.ac.uk/browse/instrument/1/facilityCycle/14/investigation/87?view=card) with this version. You should see a different color hyperlink in the header and "Name" now replaced with "Dataset" in the sub-header.

## Agile board tracking
Closes #723
